source/common/upstream/cluster_manager_impl.cc

```diff
@@ -1825,6 +1825,18 @@ ClusterManagerPtr ProdClusterManagerFactory::clusterManagerFromProto(
       http_context_, grpc_context_, router_context_, server_)};
 }
 
+absl::optional<Http::HttpServerPropertiesCache::Origin>
+getOrigin(const Network::TransportSocketOptionsConstSharedPtr& options, HostConstSharedPtr host) {
+  std::string sni(host->transportSocketFactory().defaultServerNameIndication());
```

Comment:
nit: per [go/totw/88](http://goto.google.com/totw/88) use = instead of () here.

Reply:
done